modules/ossm-member-roll-create.adoc

```diff
@@ -102,3 +101,51 @@ spec:
 ----
 +
 . Modify the default YAML to add your projects as `members`.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+[id="ossm-member-roll-create-member_{context}"]
+== Creating the {ProductName} members
+
+`ServiceMeshMember` resources can be created by service mesh users who don't have privileges to add members to the Service Mesh Member Roll directly. While project administrators are automatically given permission to create the `ServiceMeshMember` resource in their project, they cannot point it to any `ServiceMeshControlPlane` until the service mesh administrator explicitly grants access to the service mesh. Administrators can grant users permissions to access the mesh by granting them the mesh-user user role, for example: 
```

Comment:
Service Mesh Member Roll/  `ServiceMeshMemberRoll`

Administrators can grant users permissions to access the mesh by granting them the `mesh-user` user role